hoot-core-test/src/test/cpp/hoot/core/algorithms/AlphaShapeTest.cpp

```diff
@@ -61,13 +61,13 @@ class AlphaShapeTest : public HootTestFixture
 
 public:
 
-  virtual void setUp()
+  AlphaShapeTest()
   {
-    HootTestFixture::setUp();
+    setResetType(ResetBasic);
     TestUtils::mkpath("test-output/algorithms");
   }
 
-  void createDonut(vector< pair<double, double> >& p, double innerRadius, double outerRadius,
+  void createDonut(vector<pair<double, double>>& p, double innerRadius, double outerRadius,
```

Comment:
I also got rid of the extra spaces in template-of-template classes since C++11 allows for this.  So nice.